sdk/identity/azure-identity/src/samples/Azure Cache For Redis/Jedis/Azure-AAD-Authentication-With-Jedis.md

```diff
@@ -0,0 +1,207 @@
+## Azure Cache For Redis Azure AD With Jedis Client Library
+
+### Table of contents
+
+- [Jedis Library](#jedis-library)
```

Comment:
You can delete this section, since the doc's title already makes it clear the focus is Jedis.